targets/XPlatCppSdk/source/cppsdk/include/playfab/PlayFabCallRequestContainer.h

```diff
@@ -1,46 +1,35 @@
 #pragma once
 
 #include <playfab/PlayFabError.h>
+#include <playfab/PlayFabCallRequestContainerBase.h>
 
 // Intellisense-only includes
 #include <curl/curl.h>
 
+#include <unordered_map>
+
 namespace PlayFab
 {
-    class CallRequestContainerBase
-    {
-    public:
-        CallRequestContainerBase(){};
-        virtual ~CallRequestContainerBase(){};
-    };
-
-    typedef void(*RequestCompleteCallback)(CallRequestContainerBase& reqContainer);
-    typedef std::shared_ptr<void> SharedVoidPointer;
-
     /// <summary>
     /// Internal PlayFabHttp container for each api call
     /// </summary>
     class CallRequestContainer : public CallRequestContainerBase
     {
     public:
-        CallRequestContainer();
+        CallRequestContainer(const CallRequestContainerBase& reqContainer);
```

Comment:
>const CallRequestContainerBase& reqContainer) [](start = 29, length = 45)

We're making a temporary object of the base class - probably we should directly create the child class object.